packages/browser/src/node/serverTester.ts

```diff
@@ -22,9 +22,10 @@ export async function resolveTester(
     )
   }
 
-  const { contextId, testFile } = server.resolveTesterUrl(url.pathname)
-  const project = server.project
-  const state = server.state
+  const { sessionId, testFile } = server.resolveTesterUrl(url.pathname)
+  const session = server.vitest._browserSessions.getSession(sessionId)
+  // TODO: if no session, 400
+  const project = server.vitest.getProjectByName(session?.project.name ?? '')
```

Comment:
`testerHtml` is defined by `server.testerHtml` below, but doesn't it need to come from `project.config.browser.testerHtmlPath` if it can be customized per instance?

Reply:
servers are created per project, so it comes from there already

Reply:
I see that there can be many `BrowserServer`, but this `BrowserServer` is coming from vite middleware plugin, so isn't this always pointing to the first project?

https://github.com/vitest-dev/vitest/blob/9ba935d5db2769e7bc608255e0be95d89362f9ec/packages/browser/src/node/plugin.ts#L49